renovate.json

```diff
@@ -16,6 +16,12 @@
   "bumpVersion": null,
   "semanticCommitScope": null,
   "prHourlyLimit": 0,
+  "packageRules": [
```

Comment:
Can we add one more for patches perhaps?

Reply:
Or maybe we only need to group patches and keep minors separate? What do you think

Reply:
Thanks for the review @sidharthachatterjee! I'll take a look at both options and see which might be more appropriate.

Reply:
@sidharthachatterjee I reviewed recent package updates, and whilst the vast majority of updates are patches (so definitely more important to batch), grouping together minor updates also feels like it would clear up a fair amount of noise with any major risks.

Let me know if you have any other thoughts, and thank you so much for such a helpful, welcoming introduction to contributing to Gatsby. 😄

Reply:
I completely agree with you and this looks like a good approach! 

Thank you so much for taking the time to research this and to make a thought out decision and of course for contributing back this change!